clusterloader2/cmd/clusterloader.go

```diff
@@ -146,7 +146,7 @@ func completeConfig(m *framework.MultiClientSet) error {
 			klog.Errorf("Getting master internal ip error: %v", err)
 		}
 	}
-	if clusterLoaderConfig.ClusterConfig.Provider != "aks" || clusterLoaderConfig.ClusterConfig.Provider != "gke" {
+	if clusterLoaderConfig.ClusterConfig.Provider != "aks" && clusterLoaderConfig.ClusterConfig.Provider != "gke" {
```

Comment:
@alexeldeib I imagine this is what was intended? Right now this is always `true`, let me know if you wanted a different behavior here!

Reply:
nice catch, thanks!